CHANGELOG.md

```diff
@@ -114,9 +114,9 @@ generated user module:
 ### Enhancements
   * Export `createHook` from `phoenix_live_view` to allow CustomElement's to use hooks
   * Expose programmable JS command interface to hooks for showing, hiding, addClass, etc from a hook which integrates with server DOM patching
-  * Support targetting inner and closest query selectors in JS commands with `to: {:inner, "a"}` and `to: {:closet, "tr"}`, etc.
+  * Support targeting inner and closest query selectors in JS commands with `to: {:inner, "a"}` and `to: {:closet, "tr"}`, etc.
   * Throw a more informative error when `JS.dispatch/2` is used with an input event on an invalid target
-  * Validate slot options (prevents typos like `slot :myslot, requird: true`)
+  * Validate slot options (prevents typos like `slot :myslot, required: true`)
```

Comment:
That is actually intentional :D

Reply:
Touché! Fixed in https://github.com/phoenixframework/phoenix_live_view/pull/3440/commits/168b619c6f0490bf005922426545089068cc814d